parsers/CO.py

```diff
@@ -45,13 +54,11 @@
     "JET-A1": "oil",
 }
 
-XM_DELAY_MIN = 2
-XM_DELAY_MAX = 5
-
 
 @refetch_frequency(timedelta(days=1))
+@use_proxy(country_code="CO", monkeypatch_for_pydataxm=True)
 def fetch_consumption(
-    zone_key: ZoneKey,
+    zone_key: ZoneKey = ZoneKey("CO"),
```

Comment:
While I think this makes sense I need to double check a few things internally before we can merge this, specifically how much bandwidth/data we have available first as they don't yet support pay as you go billing or if we need to increase this so not all parsers that use the proxy goes down.

Hopefully I can get this done tomorrow or on Friday otherwise it will be early next week.